main.go

```diff
@@ -0,0 +1,24 @@
+package main
```

Comment:
I kinda liked having the main function in cmd/kubeconform... I might move the validation to pkg/kubeconform.. still thinking about this.

Reply:
I saw different ways to make it, but if that's your preference, maybe putting the main back in `cmd/kubeconform` and just moving `validate` to its own package under `cmd/kubeconform/validate`.

Reply:
Another idea, we can just put `Validate()` under `pkg/validator/validator.go`.

Reply:
I've not completely made up my mind on this, I'm likely to try out something that would allow you to work, but I might reserve myself the right to break that location/interface for a little while while I find a good place for it :)